code-climate-rule-sets/phpcs-themes.xml

```diff
@@ -2,7 +2,13 @@
 <ruleset name="Responsive Framework Custom Ruleset">
 	<description>Custom sniff configuration specifically for Responsive Framework Child Themes.</description>
 
-	<rule ref="WordPress" />
+	<!-- Taken from WPCS 1.0.0 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/1.0.0/WordPress/ruleset.xml -->
```

Comment:
Since this is directly taken from the WPCS 1.0.0 why not just use `<rule ref="WordPress" />`?

Reply:
I could be misunderstanding, but I believe they are saying if you include just "WordPress", it includes the three subsets plus all sniffs in the Sniffs directory, which has lingering VIP sniffs that weren't completely removed yet because they needed further discussion.

From https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1440#issuecomment-408681926:
>The WordPress ruleset included the three sub-standards + every single sniff in the Sniffs directory, which is why these three will be included when you run against the WordPress standard.

For now, I'll revert back to the normal WordPress standard as you requested. This PR has good history on the alternative approach in case VIP sniffs keep creeping up.